test/test_watch_schedule_builder/test_per_week_limit.py

```diff
@@ -0,0 +1,29 @@
+from datetime import date
+from ketchup import watch_schedule_builder as wsb
+
+
+def test_per_week_limit_none():
```

Comment:
i havent run these but they look roughly correct on glance. though this test is only testing that the "is_possible" response is set to true, not that anything else occurred. So we may need a bit more depth on this test specifically?

Reply:
Yeah definitely I see what you mean. They were just baseline tests for myself, but as you said in a previous, we want to be able to test each line of code in the file.